### ISSUE TITLE
MLFlow UI crashes after executing with mlflow.start_run():

### ISSUE THREAD

**Issue by rakesh7464** [mlflow/mlflow#4012]
### System information

- Stack: Pyspark over cloudera on-prem cluster
- MLflow installed from (source or binary): binary
- MLflow version (run mlflow --version): 1.13.1
- Python version: Python 2.7.5
- Exact code that caused the issue: 

`with mlflow.start_run():

    dtc = DecisionTreeClassifier(random_state=10)
    dtc.fit(X_train, y_train)
    y_pred_class = dtc.predict(X_test)
    accuracy= metrics.accuracy_score(y_test, y_pred_class)

     print(accuracy)

     mlflow.log_param("random_state", 10)
     mlflow.log_metric("accuracy", accuracy)
     mlflow.sklearn.log_model(dtc, "model")
     modelpath = "///home/corp.tcf.biz/t283223/mlruns/model-%s-%f" % ("decision_tree", 1.1)
     mlflow.sklearn.save_model(dtc, modelpath)  
  
     mlflow.log_artifact("iris1.png")`

### Describe the problem
After running the cell above(as part of a notebook using scikit learn libraries, when we try to open mlflow ui, it gives us the error:
![image](https://user-images.githubusercontent.com/12123019/105401073-59627900-5beb-11eb-9ab9-c144f9bc8938.png)

### Developer log from chrome:

Failed to load resource: the server responded with a status of 400 (BAD REQUEST)
ActionUtils.js:73 XHR failed Object
error @ ActionUtils.js:73
RequestStateWrapper.js:72 ERROR Array(1)
$t @ RequestStateWrapper.js:72
react-dom.production.min.js:3282 Error: A request error occurred.
    at $t (RequestStateWrapper.js:73)
    at t.value (RequestStateWrapper.js:61)
    at h (react-dom.production.min.js:2740)
    at beginWork (react-dom.production.min.js:2916)
    at o (react-dom.production.min.js:4051)
    at i (react-dom.production.min.js:4068)
    at w (react-dom.production.min.js:4296)
    at _ (react-dom.production.min.js:4263)
    at b (react-dom.production.min.js:4254)
    at v (react-dom.production.min.js:4213)
_r @ react-dom.production.min.js:3282
AppErrorBoundary.js:19 Error: A request error occurred.
    at $t (RequestStateWrapper.js:73)
    at t.value (RequestStateWrapper.js:61)
    at h (react-dom.production.min.js:2740)
    at beginWork (react-dom.production.min.js:2916)
    at o (react-dom.production.min.js:4051)
    at i (react-dom.production.min.js:4068)
    at w (react-dom.production.min.js:4296)
    at _ (react-dom.production.min.js:4263)
    at b (react-dom.production.min.js:4254)
    at v (react-dom.production.min.js:4213) Object
value @ AppErrorBoundary.js:19
:9023/#/:1 Uncaught (in promise) exhr: {readyState: 4, getResponseHeader: ƒ, getAllResponseHeaders: ƒ, setRequestHeader: ƒ, overrideMimeType: ƒ, …}abort: ƒ (e)always: ƒ ()catch: ƒ (e)done: ƒ ()fail: ƒ ()getAllResponseHeaders: ƒ ()getResponseHeader: ƒ (e)overrideMimeType: ƒ (e)pipe: ƒ ()progress: ƒ ()promise: ƒ (e)readyState: 4responseJSON: {error_code: "INVALID_PARAMETER_VALUE", message: "Invalid experiment ID: 'model-decision_tree-1.000000'"}responseText: "{"error_code": "INVALID_PARAMETER_VALUE", "message": "Invalid experiment ID: 'model-decision_tree-1.000000'"}"setRequestHeader: ƒ (e,t)state: ƒ ()status: 400statusCode: ƒ (e)statusText: "BAD REQUEST"then: ƒ (e,r,o)__proto__: Object__proto__: Object
DevTools failed to load SourceMap: Could not load content for http://172.16.216.9:9023/static-files/bootstrap/bootstrap.min.css.map: HTTP error: status code 404, net::ERR_HTTP_RESPONSE_CODE_FAILURE

**Comment by harupy** [comment]
```python
modelpath = "///home/corp.tcf.biz/t283223/mlruns/model-%s-%f" % ("decision_tree", 1.1)
```
I think this line broke the `mlruns` directory.

**Comment by harupy** [comment]
Removing `model-decision_tree-1.000000` should fix the issue.

**Comment by rakesh7464** [comment]
Thanks! This worked for me.
